packages/cli/src/oclif/commands/promote.js

```diff
@@ -0,0 +1,153 @@
+const _ = require('lodash');
+const colors = require('colors/safe');
+
+const BaseCommand = require('../ZapierBaseCommand');
+const utils = require('../../utils');
+const { buildFlags } = require('../buildFlags');
+
+const serializeErrors = errors => {
+  const opener = 'Promotion failed for the following reasons:\n\n';
+  if (typeof errors[0] === 'string') {
+    // errors is an array of strings
+    return opener + errors.map(e => `* ${e}`).join('\n');
```

Comment:
it wasn't like this before, but it might look nice to indent the bullets:

```js
errors.map(e => `  * ${e}`).join('\n')
```

Same below. In fact, I might even put this on `BaseCommand` as `printList(opener, errors)`

Reply:
This is what it looks like without the indent:

![](https://zappy.zapier.com/747E06BA-273B-4787-9AAD-FB50FEEA05BB.png)

The output message is thrown with `new Error(message)` instead of printing, so `printList()` on `BaseCommand` probably isn't what we want.

And since the entire error message is already indented, so I wouldn't add another indent to the bullets.

Reply:
that's funny, the indentation relative to the parent message imparts ownership to me. It's the difference between:

```
promotion failed because:
  * a
  * b
  * c
validation failed because:
  * a
  * b
  * c
```

vs

```
promotion failed because:
* a
* b
* c
validation failed because:
* a
* b
* c
```

whereas the latter feels more clear. But, given that it's pretty clear and there's no secondary list, i'm happy either way here. 😁